examples/getting-started.md

```diff
@@ -0,0 +1,52 @@
+## Install from npm
```

Comment:
It's the getting started tutorial. It shows up a little nicer in the tutorials section. I'd like to keep it.